packages/search-input/src/SearchInput/SearchInput.story.tsx

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-console */
+import React, { ChangeEventHandler, useState } from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { startCase } from 'lodash';
+
+import { css } from '@leafygreen-ui/emotion';
+import {
+  storybookArgTypes,
+  // @ts-ignore
+  storybookExcludedControlParams,
+} from '@leafygreen-ui/lib';
+
+import { SearchInput, SearchResult, SearchResultGroup } from '..';
+
+import { State } from './SearchInput.types';
+
+export default {
+  title: 'Components/SearchInput',
+  component: SearchInput,
+  argTypes: {
+    darkMode: storybookArgTypes.darkMode,
+    state: { control: 'select', options: Object.values(State) },
+    id: { control: 'string' },
+    placeholder: { control: 'string' },
+    value: { control: 'string' },
+    disabled: { control: 'boolean' },
+    ref: { control: 'none' },
+  },
+  parameters: {
+    controls: {
+      exclude: [
+        ...storybookExcludedControlParams,
+        'value',
+        'id',
+        'showWedge',
+        'onSubmit',
+      ],
+    },
+  },
+} as ComponentMeta<typeof SearchInput>;
+
+export const Basic: ComponentStory<typeof SearchInput> = props => (
+  <SearchInput
+    className={css`
+      width: 200px;
+    `}
+    {...props}
+  />
+);
+
+export const WithResults: ComponentStory<typeof SearchInput> = props => (
+  <SearchInput
+    className={css`
+      width: 200px;
+    `}
```

Comment:
nit: we can use `<Basic>` here so we don't have to duplicate the inline styles

Reply:
Technically correct, but I feel like this makes `children` a bit more complicated